pkg/KubeArmorOperator/runtime/runtime.go

```diff
@@ -10,6 +10,8 @@ import (
 	"go.uber.org/zap"
 )
 
+// TODO: fix typos
```

Comment:
WHat's this for?

Reply:
I found a couple of typos in our operator code such as:
https://github.com/kubearmor/KubeArmor/blob/3bf31234d0927bdb5d5799e6b021d5dba4690de0/pkg/KubeArmorOperator/runtime/runtime.go#L24
https://github.com/kubearmor/KubeArmor/blob/3bf31234d0927bdb5d5799e6b021d5dba4690de0/pkg/KubeArmorOperator/runtime/runtime.go#L34
https://github.com/kubearmor/KubeArmor/blob/3bf31234d0927bdb5d5799e6b021d5dba4690de0/pkg/KubeArmorOperator/runtime/runtime.go#L40
https://github.com/kubearmor/KubeArmor/blob/3bf31234d0927bdb5d5799e6b021d5dba4690de0/pkg/KubeArmorOperator/common/defaults.go#L31

I thought to address these in a separate PR, since this PR is to add support and not to fix typos. However, if you'd prefer, I can address these in a new commit in this PR itself.

Reply:
Yup let's address it here in a separate commit. Thanks for noticing.